### PR TITLE
chore: add _rand.c to gitignore

### PR DESCRIPTION
**Pull request by Kyle-Verhoog**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -86,6 +87,7 @@ celerybeat-schedule
 # virtualenv
 venv/
 ENV/
+.venv*/
```
Comment (jd) [inline .gitignore:90]: what's used for/from?

Comment (Kyle-Verhoog) [reply .gitignore:90]: I use this convention and I thiiink it's fairly common in general

Comment (jd) [reply .gitignore:90]: /me whispers

https://julien.danjou.info/properly-managing-your-gitignore/

Comment (Kyle-Verhoog) [reply .gitignore]: fiiiine 😆

**Pull request merged**